app/assets/src/components/PipelineSampleReport.jsx

```diff
@@ -820,7 +820,12 @@ class PipelineSampleReport extends React.Component {
   // path to NCBI
   gotoNCBI(e) {
     const taxId = e.target.getAttribute("data-tax-id");
-    const ncbiLink = `https://www.ncbi.nlm.nih.gov/Taxonomy/Browser/wwwtax.cgi?mode=Info&id=${taxId}`;
+    let num = parseInt(taxId);
+    if (num < 0) {
+      num = -num % 1e8;
```

Comment:
How well does JavaScript implement % 1e8?   I think with fp64 it's important that this be `(-num) % 1e8` rather than `-(num % 1e8)`.

Reply:
Also this should be `if (num < -1e8)` unless already implied.